examples/advanced/nlp-ner/requirements.txt

```diff
@@ -0,0 +1,7 @@
+nvflare
+torch
+torchvision
+tensorboard
+transformers
```

Comment:
Need to add the license file from transformers to [3rd party folder](https://github.com/NVIDIA/NVFlare/tree/dev/3rdParty)

Reply:
added